File: java/src/main/java/constants/HttpConstant.java
package constants;

/**
 * http 相关的常量
 */
public class HttpConstant {
    public static final String HOST = "https://api.jiandaoyun.com/api";

    public static final String HOST_V2 = HOST + "/v2";

    public static final String HOST_V1 = HOST + "/v1";

    public static final String HOST_V4 = HOST + "/v4";

    /**
     * 部门相关的接口前缀
     */
    public static final String DEPT_BASE_URL = HOST_V2 + "/department/";

    /**
     * 成员相关的接口前缀
     */
    public static final String MEMBER_BASE_URL = HOST_V2 + "/user/";

    /**
     * 应用相关的接口前缀
     */
    public static final String APP_BASE_URL = HOST_V1 + "/app/";


    /**
     * 企业互联的接口前缀
     */
    public static final String CORP_COOP_BASE_URL = HOST_V4 + "/corp_coop/";

    /**
     * 角色相关的接口前缀
     */
    public static final String ROLE_BASE_URL = HOST_V2 + "/role/";

    /**
     * 角色组相关的接口前缀
     */
    public static final String ROLE_GROUP_BASE_URL = HOST_V2 + "/role_group/";

    /**
     * 表单相关的接口前缀
     */
    public static final String FORM_BASE_URL = HOST_V2 + "/app/";

    /**
     * 表单数据相关的接口前缀
     */
    public static final String FORM_DATA_BASE_URL = HOST_V4 + "/app/";

    public static final String API_KEY = "API_KEY";

    public static final String APP_ID = "APP_ID";

    public static final String WORK_FLOW_ENTRY_ID = "WORK_FLOW_ENTRY_ID";

    public static final String ENTRY_ID = "ENTRY_ID";

}

File: python/src/constants/http_constant.py
class HttpConstant:
    HOST = "https://api.jiandaoyun.com/api"

    HOST_V1 = HOST + "/v1"
    HOST_V4 = HOST + "/v4"

    # d应用相关的接口前缀
    APP_BASE_PATH = "/{version}/app/"
    # 表单相关的接口前缀
    FORM_BASE_PATH = "/{version}/app/"

    # API_KEY
    API_KEY = "API_KEY"

    APP_ID = "APP_ID"

    WORK_FLOW_ENTRY_ID = "WORK_FLOW_ENTRY_ID"

    ENTRY_ID = "ENTRY_ID"

    # 企业互联的接口
    CORP_COOP_PATH = "/{version}/corp/guest/{suffix}"

    # 部门相关的接口
    DEPT_PATH = "/{version}/corp/department/{suffix}"

    # 成员相关的接口
    MEMBER_PATH = "/{version}/corp/user/{suffix}"

    # 角色的接口前缀
    ROLE_PATH = "/{version}/corp/role/{suffix}"

    # 角色组相关的前缀
    ROLE_GROUP_PATH = "/{version}/corp/role_group/{suffix}"

    # 应用相关的接口前缀
    APP_PATH = "/{version}/app/{suffix}"

    # 表单查询的path
    FORM_PATH = '/{version}/app/entry/{suffix}'

    # 表单数据查询的path
    FORM_DATA_PATH = '/{version}/app/entry/data/{suffix}'
